assets/src/edit-story/migration/migrations/v0022_convertOverlayPattern.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function convertOverlayPattern({ pages, ...rest }) {
+  return {
+    pages: pages.map(convertPage),
+    ...rest,
+  };
+}
+
+function convertPage({ elements, backgroundOverlay, ...rest }) {
+  const hasNonTrivialOverlay =
+    backgroundOverlay &&
+    ['solid', 'linear', 'radial'].includes(backgroundOverlay);
```

Comment:
Hmm, which would be the values here for trivial overlays? Just `none`?

Reply:
Or is this for the case if there should for some odd reason be an invalid background overlay value present?